tools/bpf2c/bpf2c.cpp

```diff
@@ -51,6 +72,11 @@ main(int argc, char** argv)
                 {"--dll",
                  {"Generate code for a Windows DLL",
                   [&](std::vector<std::string>::iterator&) { type = output_type::UserPE; }}},
+#if defined(ENABLE_SKIP_VERIFY)
+                {"--no-verify",
+                 {"Skip validate code using verifier",
```

Comment:
```suggestion
                 {"Skip validating code using verifier",
```
Is there a legitimate reason to ever skip?